### PR TITLE
[Fix](executor)Fix when Fe send empty wg list to be may cause query failed.

### PR DESCRIPTION
**Pull request by wangbo**
Currently FE may send an empty workload group list to be occasionally, this may cause be's workload group is shutdown, but never be deleted.
We need fix this case in BE, when FE send emptylist , be not delete any workload group.
Add more log to location this root reason, whether FE send emtpy list or missing some workload group.
I also find that all fe could send publish request to BE, but we expect only Master FE does this.

### CODE REVIEW

**Comment by wangbo** [comment]
run buildall

File: be/src/runtime/workload_group/workload_group_manager.cpp [hunk 1]
```diff
@@ -83,9 +83,11 @@ void WorkloadGroupMgr::delete_workload_group_by_ids(std::set<uint64_t> used_wg_i
             auto workload_group_ptr = iter->second;
             if (used_wg_id.find(tg_id) == used_wg_id.end()) {
                 workload_group_ptr->shutdown();
+                LOG(INFO) << "[topic_publish_wg] shutdown wg " << tg_id;
```
Comment (yiguolei) [inline be/src/runtime/workload_group/workload_group_manager.cpp:86]: rename to wg_id

**Review by yiguolei — DISMISSED** [review]
LGTM

**Review by yiguolei — APPROVED** [review]
LGTM

**Review by xinyiZzz — APPROVED** [review]
LGTM

**Pull request merged**